src/main/resources/templates/gardens/add.html

```diff
@@ -9,59 +9,65 @@
 
 <div th:replace="fragments :: page-header"></div>
 
-<form method="post" th:action="@{/admin/gardens/add}" enctype="multipart/form-data">
+<div class="container">
```

Comment:
All these Div class containers are is putting the elements inside of a card correct?

Reply:
Yes, the container class in bootstrap sets a max-width at bootstraps breakpoints.